contracts/BlastYield.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IBlast, YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "./interfaces/IBlast.sol";
+import {IBlastPoints} from "./interfaces/IBlastPoints.sol";
+import {IERC20Rebasing, YieldMode as IERC20Rebasing__YieldMode} from "./interfaces/IERC20Rebasing.sol";
+
+/**
+ * @title BlastYield
+ * @notice This contract is a base contract for future contracts that wish to claim Blast WETH or USDB yield to inherit from.
+ * @author LooksRare protocol team (👀,💎)
+ */
+contract BlastYield {
+    address public immutable WETH;
+    address public immutable USDB;
+
+    /**
+     * @param _blast Blast precompile
+     * @param _blastPoints Blast points
+     * @param _blastPointsOperator Blast points operator
+     * @param _owner Owner of the contract
```

Comment:
we can rename this as governor since it's no longer an OwnableTwoSteps

https://docs.blast.io/building/guides/eth-yield#governor

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/cc8287718d05f286b5301a573fe6035425025c7b